ItalianTranslation_PlanetCoaster.xml

```diff
@@ -14044,7 +14040,6 @@
 		<entry key="ParkManagement_Crime_SecurityCameraAllTimeTheftsSpotted" translation="Furti rilevati (totale)"/>
 		<entry key="ParkManagement_Crime_SecurityCameraAllTimeVandalismSpotted" translation="Atti di vandalismo rilevati (totale)"/>
 		<entry key="ParkManagement_Crime_SecurityCameraCount" translation="Telecamere di sicurezza"/>
-		<entry key="ParkManagement_Crime_SecurityCameraHeading" translation="Telecamere di sicurezza"/>
 		<entry key="ParkManagement_Crime_SecurityCameraHeading" translation="Telecamere di sicurezzas"/>
```

Comment:
Qui è errato, segnatelo per il prossimo update

Reply:
Si, ho cancellato alcune righe duplicate. E parecchi spazi.
Quella riga tra l'altro l'avevo cancellata in una versione precedente.
Il problema è che se non mi fai il merge di almeno una mia pull, mi porto avanti i commit fino alla prossima patch.. E' esasperante..